### DISCUSSION TITLE
Consistency with file names

### DISCUSSION THREAD

**Discussion by yjose** [founded-labs/react-native-reusables#272] [Ideas]
Hey @mrzachnugent , thank you for the great work 

While installing React Native Reusable, I noticed that filenames are not consistent throughout the project. Inside the lib/icon, we are using uppercase files (e.g., `iconWithClassName`, `Sun`), which is not the case inside the `components/ui` folder, which uses kebab-case as the default.

I support using kebab-case for all files as its the most used format now a day in the js ecosystem.

Let me know if you are aware of this and if you would like to use kebab-case for all files, including the `lib` folder. I would be happy to open a new PR with those changes.

Btw, i tried react native reusable and i am considering adding it as default choice for our [Obytes Starter ](https://starter.obytes.com/)

**Comment by mrzachnugent** [comment]
Hey @yjose, thanks for bringing it up, it is an oversight. Files from shadcn/ui use the kebab-case naming convention whereas I use PascalCase for components and camelCase for functions in most of my other projects. The file naming convention should be standardized.

This type of change has a lot of implications for the `cli` so it'll be important that it handles updating existing projects with the current file names gracefully.

And if you decide to make it your default choice, I would love to try it out!
